client/src/catalog-app.html

```diff
@@ -194,6 +194,8 @@
       },
 
       _pageReady: function() {
+        Polymer.AppLayout.scroll(0);
```

Comment:
I think we should do this in `_setLoading` since `_pageReady` only gets called when `page` changes.